docfx/api-override/Meadow.Foundation/Meadow.Foundation.ICs.IOExpanders.Mcp3008.md

```diff
@@ -0,0 +1,66 @@
+---
+uid: Meadow.Foundation.ICs.IOExpanders.Mcp3008
+remarks: *content
+---
+
+| Mcp3008 | |
+|--------|--------|
+| Status | <img src="https://img.shields.io/badge/Working-brightgreen" style="width: auto; height: -webkit-fill-available;" alt="Status badge: working" /> |
+| Source code | [GitHub](https://github.com/WildernessLabs/Meadow.Foundation/tree/main/Source/Meadow.Foundation.Peripherals/ICs.ADCs.Mcp3xxx) |
+| Datasheet(s) | [GitHub](https://github.com/WildernessLabs/Meadow.Foundation/tree/main/Source/Meadow.Foundation.Peripherals/ICs.ADCs.Mcp3xxx/Datasheet) |
+| NuGet package | <a href="https://www.nuget.org/packages/Meadow.Foundation.ADCs.Mcp3xxx/" target="_blank"><img src="https://img.shields.io/nuget/v/Meadow.Foundation.ADCs.Mcp3xxx.svg?label=Meadow.Foundation.ADCs.Mcp3xxx" alt="NuGet Gallery for Meadow.Foundation.ADCs.Mcp3xxx" /></a> |
+
### Code Example
+
+```csharp
+Mcp3008 mcp;
+
+IAnalogInputPort port;
+
+public override Task Initialize()
+{
+    Resolver.Log.Info("Initialize");
+
+    IDigitalOutputPort chipSelectPort = Device.CreateDigitalOutputPort(Device.Pins.D01);
+
+    mcp = new Mcp3008(Device.CreateSpiBus(), chipSelectPort);
+
+    port = mcp.CreateAnalogInputPort(mcp.Pins.CH0, 32, TimeSpan.FromSeconds(1), new Voltage(3.3, Voltage.UnitType.Volts), Mcp3xxx.InputType.SingleEnded);
+
+    port.Updated += (s, result) =>
+    {
+        Console.WriteLine($"Analog event, new voltage: {result.New.Volts:N2}V, old: {result.Old?.Volts:N2}V");

```

Comment:
Probably should be a Resolver Log

Reply:
that's a Meadow.Foundation issue vs docs - but agreed